### DISCUSSION TITLE
Display as thumbnails for other element indexes

### DISCUSSION THREAD

**Discussion by membla** [craftcms/cms#12706] [Ideas] [answered]
For certain elements, a thumbnail is more appropriate than a title when presenting it and it would thus make more sense to display them in a thumbnail grid view, much like assets are presented when using the "Display as thumbnails" option.

It should be possible to use a  "Display as thumbnails" mode for entries, etc.

**Comment by brandonkelly** [comment, marked as answer]
Craft 5 added the ability to specify an Assets field as the thumbnail provider for entries, within field layout designers.
